cash_register.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+# This is the CashRegister Class
+class CashRegister
+  def initialize
+    @total = 0.00
+    two_decimal = format('%.2f', @total)
+    puts "The cash register balance is $#{two_decimal}"
+  end
+
+  def purchase(item)
+    food_items = ['Organic Bears Milk', 'Wagyu Ribeye', 'Smoked Bacon Salt']
+    food = food_items.sample(1).join
+    @total += item
+    two_decimal = format('%.2f', @total)
+    puts "You just purchased the #{food} for $#{format('%.2f', item)}. \b
```

Comment:
puts "You just purchased the #{food}, your new total is #{total}"

You just purchased the hamburger, your new total is $3.25

* make the method handle rounding and formatting